tools/training-benchmark.py

```diff
@@ -163,16 +163,17 @@ def wrapper(*args, **kwargs):
             true_energies = batch_y['energies'].to(parser.device)
             predicted_energies = []
             num_atoms = []
+            atomic_properties = []
 
             for chunk_species, chunk_coordinates in batch_x:
-                chunk_species = chunk_species.to(parser.device)
-                chunk_coordinates = chunk_coordinates.to(parser.device)
+                atomic_chunk = {'species': chunk_species.to(parser.device), 'coordinates': chunk_coordinates.to(parser.device)}
+                atomic_properties.append(atomic_chunk)
                 num_atoms.append((chunk_species >= 0).to(true_energies.dtype).sum(dim=1))
```

Comment:
`num_atoms` is still on CPU. Maybe we should do:
```
chunk_species = chunk_species.to(parser.device)
chunk_coordinates = chunk_coordinates.to(parser.device)
```
and then use the updated `chunk_species`?

Reply:
Ahh, yes, will do

Reply:
great, thanks, I am going to add a couple of notes here then merge

Reply:
Hey, why aren't tests running?

Reply:
fixed I guess